titiler/pgstac/main.py

```diff
@@ -63,15 +63,14 @@
 )
 templates = Jinja2Templates(env=jinja2_env)
 
-postgres_settings = PostgresSettings()
```

Comment:
The settings will be initialized later in the lifespan so there is no real need to do this. 

Now instead of failing on import it will fail during the startup event (if no env are found)